src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
```

Comment:
I meant to ask this before.  Why are these empty props and targets needed?  I thought the empty ones in the test folder would be sufficient.

Reply:
Good catch, I just tested it and they aren't necessary. I think I added them to the checked-in formatting project early on when I was trying to get things to work and didn't check that they were actually needed.